app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/SubscribedDetails.scala

```diff
@@ -16,9 +16,8 @@
 
 package uk.gov.hmrc.cgtpropertydisposalsfrontend.models
 
-import cats.Eq
+import cats.kernel.Eq
```

Comment:
we shouldn't need to dig into the `kernel` package I don't think

Reply:
yeah, removed